autoload.php

```diff
@@ -68,7 +68,8 @@ public function load_class( $class ) {
 
 		// work backwards through the namespace names of the fully-qualified
 		// class name to find a mapped file name
-		while ( false !== $pos = strrpos( $prefix, '\\' ) ) {
+		$pos = strrpos( $prefix, '\\' );
+		while ( false !== $pos ) {
```

Comment:
So I didn't test this but it seems like this wouldn't work the way we want anymore, by removing the `strrpos` out of the `while` statement. If I'm following this correctly, the way this is set up now, the `while` statement would either only execute once or would get stuck in an infinite loop, since the value of `$pos` never changes after it's first set.

Reply:
Thanks for the head-up, you're right, `$pos` doesn't get update because `strrpos` will keep using the value of `$prefix` outside the loop.

Reply:
@dkotter I reverted the change of this line. Tested and it works with `autoload.php`.